win32ss/user/user32/misc/imm.c

```diff
@@ -967,7 +968,7 @@ ImeWndProc_common(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam, BOOL unicod
              ERR("Wrong window class for Ime! fnId 0x%x\n",pWnd->fnid);
              return 0;
           }
-          pimeui = ((PIMEWND)pWnd)->pimeui;
```

Comment:
Can `pimeui` be removed from the `PIMEWND` structure?

Reply:
The `WND` structure can have extra bytes of `WNDCLASS[EX].cbWndExtra` bytes.
On user land, it is not safe, I think.